# message as a state machine to monitor upper motion boundary for collision avoidance
# publish for topics

bool in_action  # whether it goes into the upper area of the conveyor belt
float64 upper_boundary  # the upper boundary data


